test/.storybook/preview.js

```diff
@@ -0,0 +1,6 @@
+import {addParameters} from '@storybook/preact';
+
+addParameters({
+  localBaseUrl: "https://storage.googleapis.com/amp-test-website-1/amp_dist_82969/dist",
+  localAlternateNames: false,
+});
```

Comment:
I'm not submitting this file, but putting it up as an example of how this would look like.

(For a PR deploy bot, with unknown URLs based on build numbers, we should probably use a relative path `../../../dist`)

Reply:
Let's maybe submit it with the value for `localhost:8000`?

Also, why are there `../../../` in the `localBaseUrl` now? Is it going relative from the storybook URL format? Would that ever be useful?

Reply:
Got it. Is it better though to just provide the whole `/amp_dist_${BUILD_NUMBER}/dist` as a parameter?

Reply:
I'm mostly trying to prevent having to modify or generate configuration code during the build task on `amphtml`'s side.

Maybe we can turn this on its head and avoid configuration completely by inspecting the Storybook serve origin. That way `amphtml` doesn't have to deal with diverting configuration per build at all.

origin  | prefix options | alternate names
------------ | -------------| -------------
`https://cdn.ampproject.org` | cdn | no
`http://localhost` | cdn, `localhost/dist` | when localhost
`https://(any origin)` | cdn, `(any origin)/dist` | no

Reply:
Not sure I understood this part. Could you clarify?

Reply:
Or maybe even simpler by making the dev server the sole outlier:

origin  | prefix options | alternate names
------------ | -------------| -------------
`cdn.ampproject.org` | cdn | when localhost
`localhost:(port)` | cdn, dev server | when localhost
`(any)` | cdn, dev server, `(any origin)/dist` | when localhost

One could then toggle between a specific RTV/PR build and a local one in the same browser tab.

Reply:
Sorry, deleted my previous comments as I posted before I saw your question.

The idea is that you get different prefix options based on the URL the Storybook is being served on:

- `cdn.ampproject.org` only allows `cdn.ampproject.org`
- `localhost` allows `cdn.ampproject.org` and `localhost`
- any other origin allows `cdn.ampproject.org` and `localhost`, and can set additional `ampBaseUrlOptions` as a parameter.

Only `localhost` gets alternate names for unminified files.